web_widget_timepicker/__init__.py

```diff
@@ -0,0 +1 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
Make it a habit to turn all license lines to https://

Reply:
Done for every *.py